webpack.config.js

```diff
@@ -35,3 +29,35 @@ module.exports = {
     ]
   }
 };
+
+module.exports = [{
+  entry: {
+    "ga-lite": "./src/ga-lite"
+  },
+  optimization: {
+    minimize: false
+  },
+  ...umd
+}, {
+  entry: {
+    "ga-lite.min": "./src/ga-lite"
+  },
+  ...umd
+}, {
+  entry: {
+    "ga-lite.esm": "./src/ga-lite"
+  },
+  mode: "production",
+  output: {
+    ...output,
+    library: {
+      type: "module"
+    }
+  },
+  optimization: {
+    minimize: false
+  },
```

Comment:
I did not bother to create a minified ESM output since:

- I hardly see any ESM library does this
- User's own bundler can do this
- CDN can do it as well